purchase_reception_status/views/purchase_order.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2020 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="before">
+                <field name="reception_status" states="purchase,done" />
+                <field
+                    name="force_received"
+                    groups="purchase.group_purchase_manager"
+                    states="purchase,done"
+                />
+            </field>
+        </field>
+    </record>
+    <record id="purchase_order_tree" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree" />
+        <field name="arch" type="xml">
+            <field name="invoice_status" position="before">
+                <field name="reception_status" />
+            </field>
+        </field>
+    </record>
+    <record id="view_purchase_order_filter" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.search</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_view_search" />
+        <field name="arch" type="xml">
+            <filter name="not_invoiced" position="before">
```

Comment:
I think this filter `not_invoiced` no longer exist in Odoo 13, this yield error for me.

Reply:
Well it is there for me:
![image](https://user-images.githubusercontent.com/19169467/87012593-c8393100-c1c9-11ea-96e8-764ff9fa5973.png)

![image](https://user-images.githubusercontent.com/19169467/87012468-9b851980-c1c9-11ea-91bd-b17822d9defe.png)

Reply:
You are right.